lib/mergePR.js

```diff
@@ -0,0 +1,33 @@
+const {spawn}  = require("child_process")
+const settings = require(`${process.cwd()}/lib/config.js`);
+(async () => {
+    const username = settings.phases['build'].credentials.idir.user 
+    const password = settings.phases['build'].credentials.idir.pass
+    const url = settings.bitbucketUrl
+    const pr = settings.options.pr
+    const repoName = settings.options.repoName
+    const projectName = settings.options.projectName
+    const p1 = spawn('curl', ['-H', 'Content-type: application/json', '-u', `${username}:${password}`,'-X', 'GET', `${url}/rest/api/latest/projects/${projectName}/repos/${repoName}/pull-requests/${pr}/merge` ])
+    let p2 = spawn('jq', ['.canMerge'], {stdio: [p1.stdout, 'pipe', process.error]})
+    p2.stdout.on( 'data', data => {
+        if(`${data}`){
+            const p3 = spawn('curl', ['-H', 'Content-type: application/json', '-u', `${username}:${password}`,'-X', 'POST', `${url}/rest/api/latest/projects/${projectName}/repos/${repoName}/pull-requests/${pr}/merge?version` ])
```

Comment:
Didn't use git curl api before but should this be 'GET' not 'POST'?

Reply:
No, in this scenario, this is how we cant get the merge version using POST